incubator/ark-server/templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  namespace: {{ .Values.namespace.name }}
```

Comment:
You'll want this to be a namespace that is separate from where the backups/restores/etc are placed. In Ark v0.7.1, this is `heptio-ark-server` by default.